sdks/java/io/redis/src/test/java/org/apache/beam/sdk/io/redis/RedisIOTest.java

```diff
@@ -249,6 +249,18 @@ public void testReadBuildsCorrectly() {
     Assert.assertEquals(111, read.connectionConfiguration().port());
     Assert.assertEquals("pass", read.connectionConfiguration().auth());
     Assert.assertEquals(5, read.connectionConfiguration().timeout());
+    Assert.assertEquals(false, read.connectionConfiguration().ssl());
+  }
+
+  @Test
+  public void testReadBuildsWithSslCorrectly() {
```

Comment:
remove, we may consider a future PR to add `ConnectionConfigurationTest` but so far we have avoided to test AutoValue's configurations.